src/main/java/bio/terra/cli/command/spend/CreateProfile.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.cli.command.spend;
+
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.service.SpendProfileManagerService;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the third-level "terra spend create-profile" command. */
+@Command(
```

Comment:
If this and delete-profile are intended to be admin endpoints, should they be hidden?

Reply:
Hmm not sure. I think all the `terra spend` commands should be part of an admin/devops cut of the CLI. On the other hand, these two could be particularly disruptive if someone runs them by accident, similar to break-glass. I'll make just these two hidden on those grounds.